javascript-practice/src/index.html

```diff
@@ -10,8 +10,53 @@
 
 <body>
 	<!-- START HEADER SECTION -->
-	<header></header>
-	<!-- END HEADER SECTION -->
+	<header class="header">
+		<div class="top-nav">
+			<span class="header-search">
+				<img class="search-icon" src="assets/icons/search.svg" alt="" />
+				<img class="close-icon" src="assets/icons/close.svg" alt="">
```

Comment:
Where is alt? I do not think that, we will have 2 images here. Please check again